1.10/deploying-services/containerizers/index.md

```diff
@@ -17,67 +15,67 @@ The tables below provide a feature comparison of your containerizer choices on D
 
 ## DC/OS Features
 
-| 																				| Docker			| Original Mesos				| UCR 			| Comments |
-| --------------------------------------- | ----------- | --------------------- | --------- | -------- |
-| **Command**                             | Yes          | Yes                   | Yes      |          |
-| **Container Images** 										| Yes					| No										| Yes				|					 |
-| **Pods**																| No					| Yes										| Yes				|					 |
-| **GPUs**																| No					| Yes										| Yes				|					 |
-| **URIs**																| Yes					| Yes										| Yes				|					 |
-| **Docker Options**											| Yes					| No										| No				|					 |
-| **Force Pull**													| Yes					| No										| Yes				|	CLI only |
-| **Secrets**															| Yes					| Yes										| Yes				| Enterprise only |
-| **Debugging with exec**         				| No					| Yes										| Yes				|	CLI only |
-| **All Security Modes**									| Yes					| Yes										| Yes				| Enterprise only |
+| 																				| Docker			| UCR 			| Comments |
+| --------------------------------------- | ----------- | --------- | -------- |
+| **Command**                             | Yes         | Yes      |          |
+| **Container Images** 										| Yes					| Yes				|					 |
+| **Pods**																| No					| Yes				|					 |
+| **GPUs**																| No					| Yes				|					 |
+| **URIs**																| Yes					| Yes				|					 |
+| **Docker Options**											| Yes					| No				|					 |
+| **Force Pull**													| Yes					| Yes				|	CLI only |
+| **Secrets**															| Yes					| Yes				| Enterprise only |
```

Comment:
All instances of `Enterprise` should be `Enterprise DC/OS` - there is no product called simply Enterprise.